docs/template-helpRequest.md

```diff
@@ -0,0 +1,41 @@
+**Help Request**
+
+**Issue Title:** Help Request: [summary of your problem]
+
+**Step in the `settingUp.md` Guide / Failing Tests**
+
+*If you are setting up the project:*
+> Tell us the step in the setting up guide at which you are stuck
+
+*If you are resolving test failures:*
+> Tell us which tests are failing
```

Comment:
Let's add a miscellaneous category here, something like when the dev is trying to access the dev server or to deploy to AppSpot. Something like:

*Miscellaneous:*
> Tell us what you are trying to do

If you can break the misc stuffs to more sub-categories, you're welcome to.